Lib/test/test_email/test_email.py

```diff
@@ -1742,6 +1742,13 @@ def test_utf8_input_no_charset(self):
         teststr = '\u043a\u0438\u0440\u0438\u043b\u0438\u0446\u0430'
         self.assertRaises(UnicodeEncodeError, MIMEText, teststr)
 
+    def test_line_max_length(self):
+        teststr = '\n'.join(['This is a long', 'g' * 1000, 'line.'])
+        msg = MIMEText(teststr)
+        lines = msg.as_string()
+        for line in lines:
+            self.assertTrue(len(line) <= 998)
```

Comment:
You can use `assertLess` or `assertLessEqual` here.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLess